row_test.go

```diff
@@ -1,72 +1,250 @@
 package main
 
 import (
+	"bytes"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"reflect"
+	"runtime"
+	"strconv"
+	"strings"
 	"testing"
+
+	"9fans.net/go/plan9"
+	"9fans.net/go/plan9/client"
+	"github.com/rjkroege/edwood/internal/dumpfile"
 )
 
-const shortFile = `/Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood
-/mnt/font/GoRegular/13a/font
-`
+func TestRowLoad(t *testing.T) {
+	if runtime.GOOS == "windows" {
```

Comment:
it doesn't work yet?

Reply:
No, there are some assumptions in the fsys testing code, which needs to be fixed. In windows, the file server listens on a tcp connection, not $NAMESPACE/acme unix socket.